### PR TITLE
cleans up some light step code and nerfs medical wrench

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
light step code now does a switchy thing instead of an else if thing because i saw it in code and decided to betterer it
medical wrenches now cost 20 instead of the exorbitant 100 moneys

## Why It's Good For The Game
code cleanup and less dumb prices

## Changelog
:cl:
tweak: medical wrench costs 20 moneys
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
I gotta say I don't think this trucks just because you're putting two very different things in the same PR. Also, that's not a nerf, it's a buff.

File: code/game/objects/items/tools/wrench.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	force = 2 //MEDICAL
 	throwforce = 4
 
-	attack_verb = list("wrenched", "medicaled", "tapped", "jabbed", "whacked")
+	attack_verb = list("healed", "medicaled", "tapped", "poked", "analyzed") //"cobbyed"
```
Comment (Quartich) [inline code/game/objects/items/tools/wrench.dm]: `//"cobbyed"`

Nice

Comment (carlarctg) [reply code/game/objects/items/tools/wrench.dm]: if the cobbyed message plays it deals 10 liver damage

**Comment by Fikou** [comment]
medical wrench is a premium meme item, making it cost less is poopoo

**Comment by carlarctg** [comment]
bad argument

**Comment by actioninja** [comment]
this is going to wreck balance, and I will blame you when it happens.

**Pull request merged**